integration/admin-client/src/main/java/org/keycloak/admin/client/resource/GroupsResource.java

```diff
@@ -66,7 +66,8 @@
     @Consumes(MediaType.APPLICATION_JSON)
     List<GroupRepresentation> groups(@QueryParam("search") String search,
```

Comment:
This will break for existing users and we'll need an overloaded method to add the new full param.

Reply:
Thanks for reviewing. Can you give me more information ? users are never involce in the merge request I'm not sure to understande, it's just a param for a boolean who was already existing but false by default, there is no logic or any minor change in the code, just this boolean to get the full representation of a group ( so including atttributes but no mention to users in the function `toRepresentation`). Thanks :)

Reply:
Also what do you mean by an overload method ? I think I added it in GroupsResource ( line 70 ), I took inspiration from UsersResource

Reply:
As you've changed the signature for GroupsResource.groups it will be a breaking change for anyone that is using this method today. So basically what you need to do is to add another method with the additional field.

Reply:
Ok I think I understand. I added a separated method in the GroupsResource interface so, also rebased on master.